consensus/src/execution_ctx.rs

```diff
@@ -298,18 +298,7 @@ impl<'a, DB: Database, T: Operations + 'static> ExecutionCtx<'a, DB, T> {
                     msg_step,
                 );
 
-                spawn_cast_vote(
-                    &mut self.iter_ctx.join_set,
-                    Arc::new(Mutex::new([0u8; 32])),
-                    candidate.clone(),
-                    self.round_update.pubkey_bls.clone(),
-                    self.round_update.clone(),
-                    msg_step,
-                    self.outbound.clone(),
-                    self.inbound.clone(),
-                    self.executor.clone(),
-                    topic,
-                );
+                // TODO: Verify
```

Comment:
NB: The concept of `vote for former candidate` should be revised when https://github.com/dusk-network/rusk/issues/1109 is implemented.

Reply:
Either way, we should keep "process_past_events"-related code to allow "relaxed" iterations